$OpenBSD$

Now b.root-servers.net is 192.228.79.201. (SERIAL 2004012900)
Now j.root-servers.net is 192.158.128.30. (SERIAL 2002110501)
Remove a.root-servers.net [198.41.0.4] for avoid overload.

--- dnsroots.global.orig	2004-02-01 17:07:17.000000000 +0900
+++ dnsroots.global	2004-02-01 17:14:46.000000000 +0900
@@ -1,5 +1,4 @@
-198.41.0.4
-128.9.0.107
+192.228.79.201
 192.33.4.12
 128.8.10.90
 192.203.230.10
@@ -7,7 +6,7 @@
 192.112.36.4
 128.63.2.53
 192.36.148.17
-198.41.0.10
+192.58.128.30
 193.0.14.129
 198.32.64.12
 202.12.27.33
